entity.go

```diff
@@ -82,3 +82,24 @@ func (e *Entity) ComponentFast(c Component) interface{} {
 func (e *Entity) ID() string {
 	return e.id
 }
+
+// Duplicate populates an array with the specified amounts of clones from an Entity.
+// NOTE: If you provide your components as pointers, the pointer will be duplicated.
+// Hence, the entities will "share" a component.
```

Comment:
Can we even add them as non-pointers? 

I mean, I don't think you can edit the values if you add them as non-pointers. Meaning everyone will add them as pointers at some point. (pun not intended)